### DISCUSSION TITLE
Request for Assistance: Latest Error with Fast-Dreambooth

### DISCUSSION THREAD

**Discussion by ParkHangah** [TheLastBen/fast-stable-diffusion#2957] [Q&A]
I’ve been using fast-dreambooth successfully, but now, it runs smoothly until the very last step, where it unexpectedly fails and an error occurs. I posted an issue about the error on the issue board(([/issues/2956])) and encountered that issue, which I have reported to Google.

however, Given that Fast-Dreambooth was updated two weeks ago, I would greatly appreciate it if you could also take a look to see if it might be related.

Thank you in advance for your time and assistance.

**Comment by TheLastBen** [comment]
are you using the latest notebook ?
